### PR TITLE
Bump outdated @since tags to 1.3.0

### PR DESCRIPTION
**Pull request by mlobofeedzai**
All the tags that have been created since 1.2.1

### CODE REVIEW

File: openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OAlgoUtils.java [hunk 1]
```diff
@@ -19,28 +19,23 @@
...
 
-import static java.util.Collections.emptyList;
-
 /**
  * Abstract class to parse H2O algorithm params, validate them and train the model.
  *
  * @param <T> The concrete type of {@link ModelParametersSchemaV3 algorithm params}.
  * @param <M> The concrete type of {@link ModelBuilder H20 Model Builder}.
- * @since 0.1.0
+ * @since 1.3.0
```
Comment (shenggwang) [inline openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OAlgoUtils.java:38]: revert this change. This class already existed since 0.1.0. Which is correct

File: openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OAlgoUtils.java [hunk 2]
```diff
@@ -108,8 +103,6 @@ protected final T parseParams(final Frame trainingFrame,
      *
      * @param h2oParams concrete implementation of {@link ModelParametersSchemaV3 algorithm params}.
      * @return concrete implementation of {@link ModelBuilder H20 Model Builder} created from the h2oParams.
-     *
-     * @since 1.2.1
```
Comment (shenggwang) [inline openml-h2o/src/main/java/com/feedzai/openml/h2o/algos/AbstractH2OAlgoUtils.java:112]: revert this deletion and below one that state since 1.2.1.
the reason is that these methods already existed in 1.2.1, they weren't introduced in 0.1.0

**Review by shenggwang — CHANGES_REQUESTED** [review]

**Review by antssilva96 — APPROVED** [review]
